test/appium/tests/base_test_case.py

```diff
@@ -67,7 +67,7 @@ def get_capabilities_sauce_lab():
     caps['appium:app'] = 'sauce-storage:' + test_suite_data.apk_name
     caps['appium:deviceName'] = 'Android GoogleAPI Emulator'
     caps['appium:deviceOrientation'] = 'portrait'
-    caps['appium:platformVersion'] = '10.0'
+    caps['appium:platformVersion'] = '12.0'
```

Comment:
@yevh-berdnyk : just curious, do we have `platformVersion` 14 available at appium/saucelabs ?

Reply:
Yes, I've just ran one of the tests on Android14 and it passed